pom.xml

```diff
@@ -401,6 +401,12 @@
             <version>${logback.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Extra runtime dependencies make running the doclet in a Javadoc context way more complex than it is now. In fact this is why the (Apache licensed) PlantUML is 'baked in'.